nav-app/src/app/viewmodels.service.ts

```diff
@@ -1676,7 +1668,7 @@ export class LayoutOptions {
 
     public _countUnscored: boolean = false;
     public set countUnscored(newval: boolean) { this._countUnscored = newval; }
-    public get countUnscored(): boolean { return this._countUnscored; }
+    public get countUnscored(): boolean { return (this.aggregateFunction === "average") ? this._countUnscored : false; }
```

Comment:
Nice use of set/get functions! I now see I didn't need to introduce the concept in my other review comment 😛